pydra/engine/helpers.py

```diff
@@ -544,3 +508,37 @@ def get_available_cpus():
 
     # Last resort
     return os.cpu_count()
+
+
+def load_and_run(task_pkl, ind=None, rerun=False, **kwargs):
+    """
+     loading a task from a pickle file, settings proper input
+     and running the task
+     """
+    task = load_task(task_pkl=task_pkl, ind=ind)
+    task._run(rerun=rerun, **kwargs)
+
+    if not task.result():
+        raise Exception("Something went wrong")
+    return task
```

Comment:
shouldn't this return a result pickle file, not task?

Reply:
it returns `task` after running it. it's likely taht doesn't have to return anything

Reply:
still returning task though, not result. i still think this should return a pointer to a pickled result file. and the result file should have a result object that indicates that the task has crashed.

Reply:
i was just double checking this, returned object is not used anywhere, so I'm removing this for now. it really could be anything here...

Reply:
actually I could put the file name, so could be used when function is used outside worker or submitter

Reply:
i think it's ok to return the result file name. however see the previous comment about improving what load and run traps.